Numpy/README.md

```diff
@@ -0,0 +1,17 @@
+# Data-Science-Content-English
+Outline For Numpy
+
+Intro to Numpy
+What is Numpy
+Why Numpy    
+  2-D and Multi-Dimension Array in Numpy
```

Comment:
List down all the methods you'll be explaining in Numpy like, dtype reshape and all.